lib/core/src/animation/animator/fixed_step.rs

```diff
@@ -42,16 +47,16 @@ impl IntervalCounter {
 // =============================
 
 struct FixedStepAnimatorData {
-    closure : Box<dyn FnMut(f32)>,
-    counter : IntervalCounter
+    callback : Box<dyn AnimationCallback>,
+    counter  : IntervalCounter
 }
 
 impl FixedStepAnimatorData {
-    pub fn new<F:AnimationCallback>(steps_per_second:f32, f:F) -> Self {
-        let closure          = Box::new(f);
+    pub fn new<F:AnimationCallback>(steps_per_second:f64, f:F) -> Self {
+        let callback         = Box::new(f);
         let step_duration    = 1000.0 / steps_per_second;
         let counter          = IntervalCounter::new(step_duration);
-        Self { closure,counter }
+        Self { callback,counter }
```

Comment:
This is a very very minor thing, but lets use a single style - `{` is like `(` and `}` is like `)` , so instead of `{ callback,counter }` wer just write `{callback,counter}`.